src/SourceCode.Chasm/IO/ChasmRepository.Tree.cs

```diff
@@ -54,36 +51,6 @@ partial class ChasmRepository // .Tree
             return dict;
         }
 
-        public virtual async ValueTask<TreeNodeMap> ReadTreeAsync(string branch, string commitRefName, CancellationToken cancellationToken)
```

Comment:
No. See comments at top of the original file. For smarter repos, such as Cosmos or SQL, we will delegate this entire operation to repo-side. So the interface and concretes need to stay here.